docs/concepts.md

```diff
@@ -322,6 +322,66 @@ represented by a message containing two Channels handles:
     connection itself; this represents the actual authority of the caller.
   - an empty integrity component.
 
+### Signature labels
+
+Oak Wasm modules can be signed using [Ed25519](https://ed25519.cr.yp.to/)
+scheme. Such signatures may be created by an Oak module reviewer who then
+publish corresponding public keys on a public resource. By signing a module
```

Comment:
The main thing is that they publish the signatures. Then their public key is also assumed to be published already.

Reply:
Updated `publishes signatures and corresponding public keys`